mtp_noms_ops/apps/security/constants.py

```diff
@@ -21,7 +21,11 @@
 
 CHECK_DETAIL_FORM_MAPPING = {
     'decision_reason': _('Give further details (optional)'),
+    'auto_accept_reason': _('Give reason for automatically accepting'),
     'rejection_reasons': dict(
         tuple(CHECK_REJECTION_CATEGORY_TEXT_MAPPING.items()) + tuple(CHECK_REJECTION_CATEGORY_BOOLEAN_MAPPING.items())
     )
 }
+
+CHECK_AUTO_ACCEPT_UNIQUE_CONSTRAINT_ERROR = \
```

Comment:
could we leave a comment to note that this is a magic constant referencing the api?